Content.Shared/_DV/Body/Systems/CPRSystem.cs

```diff
@@ -0,0 +1,97 @@
+using Content.Shared._DV.Body.Components;
+using Content.Shared._DV.Body.Events;
+using Content.Shared.DoAfter;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Popups;
+using Content.Shared.Verbs;
+
+namespace Content.Shared._DV.Body.Systems;
+
+public sealed class CPRSystem : EntitySystem
+{
+    [Dependency] private readonly MobStateSystem _mobStateSystem = default!;
+    [Dependency] private readonly SharedDoAfterSystem _doAfterSystem = default!;
+    [Dependency] private readonly SharedPopupSystem _popupSystem = default!;
+    private readonly float _cprTime = 10f;
```

Comment:
Make into `const`
Also may be worth considering replacing magic constant number with a property on `CanDoCPRComponent`?
```suggestion
    private const float CPRTime = 10f;
```

Reply:
Fair reasoning